README.md

```diff
@@ -3,32 +3,31 @@
 
 # Eclipse Che plugin registry
 
-## Build Eclipse Che plugin registry docker image
+## Build Eclipse Che plugin registry container image
+
+Most of the time you won't need to rebuild the image because we build ```quay.io/openshiftio/che-plugin-registry:latest``` after every commit in master. In case you needed to change the content of the registry (e.g. add or modify some plugins meta.yaml) you can build your own image executing
 
-Execute
 ```shell
-docker build --no-cache -t eclipse/che-plugin-registry .
+docker build --no-cache -t quay.io/openshiftio/che-plugin-registry .
```

Comment:
why not quay.io/eclipse-che/che-plugin-registry ?

Reply:
Because we currently do not push to `eclipse-che` yet

Reply:
Is this temporary if I understood correctly?

Reply:
It was not temporary (we have been pushing to quay.io/openshiftio for ~1 year) but the README hasn't been updated and refers to a few months old image (the one on DockerHub). That's the goal of this PR.

And yes we will change the image and start pushing to quay.io/eclipse-che (next week I hope) but we need not only to update the README but the cico script in this repo as well.